install/README.md

```diff
@@ -8,28 +8,61 @@ To get started with Knative, you need a Kubernetes cluster. If you aren't
 sure which Kubernetes platform is right for you, see
 [Picking the Right Solution](https://kubernetes.io/docs/setup/pick-right-solution/).
 
-We provide information for installing Knative on
-[Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/), [IBM Cloud Kubernetes Service](https://www.ibm.com/cloud/container-service), [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/), [Minikube](https://kubernetes.io/docs/setup/minikube/), [OpenShift](https://github.com/openshift/origin) and [Pivotal Container Service](https://pivotal.io/platform/pivotal-container-service) clusters.
+We provide information for installing Knative on multiple different platforms.
 
 ## Installing Knative
 
+There are several options when installing Knative:
+
+* **Bundle install** -- Comes with the default versions of all Knative
+  components. Quickest option for setup.
+
+* **Limited install** -- Installs a subset of Knative components.
+
+* **Custom install** -- Takes longer, but allows you to choose exactly which
+  components to install.
+
+For new users, we recommend the bundle install to get you up and running
+quickly.
+
+### Install guides
+
 Follow these step-by-step guides for setting up Kubernetes and installing
-Knative components on the following platforms:
+Knative components.
 
+**Bundle install guides**
+The guides below show you how to create a Kubernetes cluster with the right
+specs for Knative on your platform of choice, then walk through installing all
+available Knative components.
 * [Knative Install on Azure Kubernetes Service](Knative-with-AKS.md)
 * [Knative Install on Gardener](Knative-with-Gardener.md)
 * [Knative Install on Google Kubernetes Engine](Knative-with-GKE.md)
 * [Knative Install on IBM Cloud Kubernetes Service](Knative-with-IKS.md)
-* [Knative Install on Minikube](Knative-with-Minikube.md)
-* [Knative Install on OpenShift](Knative-with-OpenShift.md)
-* [Knative Install on Minishift](Knative-with-Minishift.md)
 * [Knative Install on Pivotal Container Service](Knative-with-PKS.md)
 
-If you already have a Kubernetes cluster you're comfortable installing
-*alpha* software on, use the following instructions:
+If you already have a Kubernetes cluster you're comfortable installing *alpha*
+software on, use the following guide to install all Knative components:
 
 * [Knative Install on any Kubernetes](Knative-with-any-k8s.md)
 
+**Limited install guides**
+The guides below install some of the available Knative components, sometimes
+without all available capabilities, to minimize disk use.
+* [Knative Install on Minikube](Knative-with-Minikube.md)
```

Comment:
Looking at the files again, it seems like the "release-lite.yaml" file that MiniKube steps use, only allows you the ability to retrieve metrics using Grafana and Prometheus (or at least installs services and adds roles). We need validation from the observability team.

Otherwise, this "install package" is basically a "bundle of Serving and Build w/o the ELK stack".